command/ca/provisioner/provisioner.go

```diff
@@ -72,3 +88,304 @@ $ step ca provisioner remove max@smallstep.com --kid 1234 --ca-config ca.json
+	}
+	azureResourceGroupFlag = cli.StringSliceFlag{
+		Name: "azure-resource-group",
+		Usage: `The Microsoft Azure resource group <name> used to validate the identity tokens.
+Use the flag multiple times to configure multiple resource groups`,
+	}
+	removeAzureResourceGroupFlag = cli.StringSliceFlag{
+		Name: "remove-azure-resource-group",
+		Usage: `Remove a Microsoft Azure resource group <name> used to validate the identity tokens.
+Use the flag multiple times to configure multiple resource groups`,
+	}
+	azureSubscriptionIDFlag = cli.StringSliceFlag{
+		Name: "azure-subscription-id",
+		Usage: `The Microsoft Azure subscription <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple subscription IDs`,
+	}
+	removeAzureSubscriptionIDFlag = cli.StringSliceFlag{
+		Name: "remove-azure-subscription-id",
+		Usage: `Remove a Microsoft Azure subscription <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple subscription IDs`,
+	}
+	azureObjectIDFlag = cli.StringSliceFlag{
+		Name: "azure-object-id",
+		Usage: `The Microsoft Azure AD object <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple object IDs`,
+	}
+	removeAzureObjectIDFlag = cli.StringSliceFlag{
+		Name: "remove-azure-object-id",
+		Usage: `Remove a Microsoft Azure AD object <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple object IDs`,
+	}
+	gcpServiceAccountFlag = cli.StringSliceFlag{
+		Name: "gcp-service-account",
+		Usage: `The Google service account <email> or <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple service accounts.`,
+	}
+	removeGCPServiceAccountFlag = cli.StringSliceFlag{
+		Name: "remove-gcp-service-account",
+		Usage: `Remove a Google service account <email> or <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple service accounts.`,
+	}
+	gcpProjectFlag = cli.StringSliceFlag{
+		Name: "gcp-project",
+		Usage: `The Google project <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple projects`,
+	}
+	removeGCPProjectFlag = cli.StringSliceFlag{
+		Name: "remove-gcp-project",
+		Usage: `Remove a Google project <id> used to validate the identity tokens.
+Use the flag multiple times to configure multiple projects`,
+	}
+	instanceAgeFlag = cli.DurationFlag{
+		Name: "instance-age",
+		Usage: `The maximum <duration> to grant a certificate in AWS and GCP provisioners.
+A <duration> is sequence of decimal numbers, each with optional fraction and a
+unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns",
+"us" (or "µs"), "ms", "s", "m", "h".`,
+	}
+	iidRootsFlag = cli.StringFlag{
+		Name: "iid-roots",
+		Usage: `The <file> containing the certificates used to validate the
+instance identity documents in AWS.`,
+	}
```

Comment:
This is currently not supported, and if we want to support it as it is it should be the file on the ca server containing the certificates used to ...